addon/types/column.js

```diff
@@ -46,5 +46,13 @@ export default EmberObject.extend({
 
   reset() {
     this.set('filters', []);
+  },
+
+  serialize() {
```

Comment:
implement the `toJSON` could be a better option https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#la_fonction_tojson()